package.json

```diff
@@ -19,6 +19,7 @@
     "dotenv": "^16.0.0",
     "express": "^4.18.0",
     "google-auth-library": "^8.0.1",
+    "jwt-decode": "^3.1.2",
```

Comment:
we cant add packages/dependencies without talking to the group/teams -- think of pckages as a very last resort

Reply:
This one is necessary to decode the response that Google provides and access the user info.